### PR TITLE
Add vision_opencv to the list of repositories.

### PR DESCRIPTION
**Pull request by clalancette**
Also do a minor bit of rearrangement for an alphabetical
listing.

connects to ros2/ros2#342

### CODE REVIEW

File: ros2.repos [hunk 1]
```diff
@@ -47,14 +47,14 @@ repositories:
     type: git
     url: https://github.com/ros/class_loader.git
     version: ros2
+  ros/urdfdom_headers:
```
Comment (mikaelarguedas) [inline ros2.repos]: I think that the place in the file was good, it should be cloned in the ros2 folder though, like the other forked repositories.

**Comment by mikaelarguedas** [comment]
This depends on https://github.com/ros2/vision_opencv/pull/1 and should thus be merged after

File: ros2.repos (hunk 1 above)
Comment (clalancette) [inline ros2.repos]: Oh, good point.  I missed that it was forked, and not the original.  I'll move it back and put it in the ros2 subfolder.

**Comment by clalancette** [comment]
Yeah, definitely.  I'm fairly confident in that one now (I'm just going to wait until it finishes the last CI), so I figured I would get this queued up.  Thanks for looking :).

**Comment by clalancette** [comment]
All right, I merged up ros2/vision_opencv, so I'll merge this up now.

**Pull request merged**